src/modules/ims_registrar_pcscf/ims_registrar_pcscf_mod.c

```diff
@@ -142,15 +143,15 @@ inline void pcscf_act_time()
  * Exported functions
  */
 static cmd_export_t cmds[] = {
-	{"pcscf_save",     		(cmd_function)w_save,                   1,  	save_fixup2,            0,ONREPLY_ROUTE },
+	{"pcscf_save",     		        (cmd_function)w_save,                   1,  	save_fixup2,            0,ONREPLY_ROUTE },
```

Comment:
Please align the formatting to the existing code, there seems to be some whitespaces/tabs here and also two more places below.

Reply:
Added some documentation for save_pending function